development.ini.sample

```diff
@@ -135,6 +139,11 @@ meetings.timezone = America/Denver
 meetings.default_duration = 7200
 meetings.icalendar.prodid = -//Mines ACM//web//EN
 
+# Custom Assets Configuration
+custom_assets.dir = /home/lpetty/mozzarella/assets
```

Comment:
should be commented out by default, as this directory definately wont exist for anyone but you.